### DISCUSSION TITLE
Mail queue not utilized by default

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1869] [General]
In a hybrid site, setting the core mail provider to Vanjaro seems to work as expected and I don't see a reason to use Core. But what surprises me is that mail being generated from a Live Form (DNN) never hits the mail queue. That leads me to believe that unless the queue is requested, the default behavior for messaging is not to queue. I was hoping that "DNN Mail" would finally gain a mail queue by using the Vanjaro mail provider.

**Comment by mksmanish79** [comment]
Mandeeps Live Form mails are added to mandeeps common mailqueue table. Vanjaro Form will run on vanjaro mailqueue.
